src/frontend/yarn.lock

```diff
@@ -1332,7 +1332,7 @@
     "@codemirror/view" "^0.19.0"
     "@lezer/common" "^0.15.0"
 
-"@codemirror/rangeset@^0.19.0", "@codemirror/rangeset@^0.19.5":
```

Comment:
This file should not be committed in this PR. It is managed in #1719

Reply:
Ok, is it just this line? I can see lot of lines related to codemirror that are updated.

Reply:
No the entire file should be removed from your PR